lib/model/message.dart

```diff
@@ -60,14 +60,12 @@ class MessageStoreImpl with MessageStore {
     }
   }
 
-  void dispose() {
-    // When a MessageListView is disposed, it removes itself from the Set
-    // `MessageStoreImpl._messageListViews`. Instead of iterating on that Set,
-    // iterate on a copy, to avoid concurrent modifications.
-    for (final view in _messageListViews.toList()) {
-      view.dispose();
-    }
-  }
+  // No `dispose` method, because there's nothing for it to do.
```

Comment:
nit:

    PerAccountStore shouldn't be an owner of the MessageListView elements.

A MessageListView isn't an "element"; that term has a specific meaning in Flutter:
https://main-api.flutter.dev/flutter/widgets/Element-class.html
and is also used (following math terminology) for an item in a collection data structure, like a List, but it's not a general synonym for "object" or "value".

Instead of "elements", "objects" would be accurate.